### DISCUSSION TITLE
S100105 - Everything works except buttons

### DISCUSSION THREAD

**Discussion by RetroVape** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#392] [Q&A]
Sorry if this is obvious or I missed something. Should the buttons on the control panel work with this installed? They make the new beep sound, but that’s it.

Controlling everything within the web interface works. The led panel shows everything it should. The buttons light up when on, just can’t control them from the panel itself.

this is a 6 pin, and all enabled buttons are 1 I’m BWC_const.h

EDIT: Nevermind, I didn’t realize that single presses are what do nothing. You have to hold the button for a short while. Everything works! 

Thanks for your hard work!

**Comment by booyakafoo** [comment]
Sorry to bug you.  I'm looking to purchase a 6pin premade for my bestway saluspa s100105.  Curious if you would be interested.  Thanks.
